ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -0,0 +1,590 @@
+  'use. You can select which version of the CDR you wish to query in this Workspace.',
+  billingAccount: 'Throughout this period of testing and development, your use of the Workbench ' +
+  'is being funded by the National Institutes of Health. In the future researchers may be ' +
+  'required to enter billing account information to cover the cost of computing time in the cloud.',
+  researchPurpose: 'You  are required to describe your research purpose, or the reason why you ' +
+  'are conducting this study. This information, along with your name, will be posted on the ' +
+  'publicly available All of Us website (https://www.researchallofus.org/) to inform our ' +
+  'participants and other stakeholders about what kind of research their data is being used for.',
+  reviewRequest: 'If you are concerned that your research may be stigmatizing to a particular ' +
+  'group of research participants, you may request a review of your research purpose by the All ' +
+  'of Us Resource Access Board (RAB). The RAB will provide feedback regarding potential for ' +
+  'stigmatizing specific groups of participants and, if needed, guidance for modifying your ' +
+  'research purpose/scope. Even if you request a review, you will be able to create a Workspace ' +
+  'and proceed with your research.',
+  underservedPopulation: 'A primary mission of the All of Us Research Program is to include ' +
+  'populations that are medically underserved and/or historically underrepresented in biomedical ' +
+  'research or who, because of systematic social disadvantage, experience disparities in health. ' +
+  'As a way to understand how much research is being conducted on these populations, All of Us ' +
+  'requests that you mark all options for underserved populations that will be included in your ' +
+  'research.'
+};
+
+
+const styles = reactStyles({
+
+  header: {
+    fontWeight: 600,
+    lineHeight: '24px',
+    color: '#262262'
+  },
+
+  requiredText: {
+    fontSize: '13px',
+    fontStyle: 'italic',
+    fontWeight: 400,
+    color: '#4A4A4A',
+    marginLeft: '0.2rem'
+  },
+
+  text: {
+    fontSize: '13px',
+    color: '#4A4A4A',
+    fontWeight: 400,
+    lineHeight: '24px'
+  },
+
+  textInput: {
+    width: '20rem',
+    borderColor: 'rgb(151, 151, 151)',
+    borderRadius: '6px',
+    marginRight: '20px',
+    marginBottom: '5px'
+  },
+
+  infoIcon: {
+    height: '16px',
+    marginLeft: '0.2rem',
+    width: '16px'
+  },
+  select: {
+    display: 'inline-block',
+    verticalAlign: 'middle',
+    position: 'relative',
+    overflow: 'visible',
+    width: '11.3rem',
+    marginRight: '20px'
+  },
+  textArea: {
+    width: '100%',
+    marginTop: '0.5rem',
+    height: '10em',
+    resize: 'none',
+    padding: '0.1rem 0.2rem',
+    background: '#fff',
+    border: '1px solid #ccc',
+    color: '#000',
+    borderRadius: '.125rem'
+  },
+  shortDescription: {
+    color: '#262262',
```

Comment:
Optional Nit for all colors: You may be able to use `app/styles/colors.ts`? (We don't do that very much in the codebase right now, but I'd love to move in that direction)